torch/_inductor/kernel/flex_attention.py

```diff
@@ -104,59 +104,40 @@ def build_subgraph_buffer(
         args: The args that are passed into the subgraph. Contains both fixed and lifted inputs.
         subgraph: The Subgraph ir for which to produce the output node
     """
-    cnt = 0
-    env = {}
-    for node in subgraph.graph_module.graph.nodes:
-        # There are two classes of placeholder inpts that we need
-        # to handle differently. For the first n_scalar_inps inputs
-        # we expect that these placeholders were generated by the make_fx call
```

Comment:
why remove doc block?

Reply:
at the least can we add some more description to `PointwiseSubgraphLowering`

Reply:
I see, this doc block shouldn't really be here lol.

Reply:
Actually, I think this doc block doesn't make sense in this function, since this function doesn't depend on that logic. If anything, it should be at where the inputs are created at the call-sites. But I think it's a bit weird to duplicate this comment there.